dao-utils/src/test_env.rs

```diff
@@ -121,7 +121,7 @@ impl TestEnvState {
     pub fn new() -> TestEnvState {
         let mut genesis_config = DEFAULT_GENESIS_CONFIG.clone();
         let mut accounts: Vec<Address> = Vec::new();
-        for i in 0..3 {
+        for i in 0..5 {
```

Comment:
In my PR i set it to 10 :D
Maybe get_accounts can get an arbitrary number and generate accounts when needed?

Reply:
If you need 10 accounts let's simply override the value in your pr. Generally, I would set a fixed number of accounts. I think it makes the setup cleaner and more predictable if all the tests use the same config.